File: Java/Car.java
package Java;

class Car {
    // Properties
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    // Methods
    void printData() {
        System.out.println("License: " + license + "\nName Driver: " + driver.name + "\nPassenger: " + passenger);
    }

    // Constructor
    Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }
}

File: JavaScript/Car.js
/* Syntax prototype
function Car(license, driver) {
    this.id;
    this.license = license;
    this.driver = driver;
    this.passenger;
}

Car.prototype.printData = function () {
    console.log("License: " + this.license
        + "\nDriver name: " + this.driver.name
        + "\nDriver document: " + this.driver.document);
} */

// Syntax class of ES6
class Car {
    constructor(license, driver) {
        this.id;
        this.license = license;
        this.driver = driver;
        this.passenger;
    }

    printDataCar() {
        console.log("License: " + this.license
            + "\nDriver name: " + this.driver.name
            + "\nDriver document: " + this.driver.document);
    }
}